changelogs/1.31.0.yaml

```diff
@@ -535,6 +535,9 @@ new_features:
     QUIC stream reset error code will be added to transport failure reason.
     This behavior can be reverted by setting the runtime flag ``envoy.reloadable_features.report_stream_reset_error_code``
     to ``false``.
+- area: basic_auth
+  change: |
+    Added support to provide an override authentication header to load Basic Auth credential.
```

Comment:
Please add the new api field link here. You can get lots of example in the change log.

Reply:
fixed